lib/types/form-types.ts

```diff
@@ -60,6 +70,8 @@ export interface ElementProperties {
   autoComplete?: string;
   dateFormat?: string;
   conditionalRules?: ConditionalRule[];
+  full?: boolean;
```

Comment:
Calling this "full" doesn't given us enough context when working with a form element -- 

Can we add this onto the `AddressComponents` 

Thinking it would be nicer to access all via the address properties under something like properties.address.x

Maybe instead of `full` it would be a boolean for splitAddress = true or false 

if (
            question?.type === FormElementTypes.addressComplete &&
            question.properties.splitAddress === false
          ) {
          
          ... split into the pieces

Reply:
Would changing the language to `combined` vs.`split` help? 

Also a thought -- could the component be made simpler by having less options? So that verified address is always full and combined for example? Just food for thought.

Reply:
I re-used a property from another field, but probably good to not do that. I'll split off into AddressComponents.

Reply:
> Would changing the language to `combined` vs.`split` help?
> 
> Also a thought -- could the component be made simpler by having less options? So that verified address is always full and combined for example? Just food for thought.

It would simpliify it - but the question I guess I'd have is "What do ours users need/want?" I'm not sure. :)

Reply:
Resolved through design change that will be occuring in a later PR.

Reply:
full?: boolean;

> Resolved through design change that will be occuring in a later PR.

Please create an issue so we can track this back.

Reply:
```suggestion
```

Can we just remove this ? Where is it being used at this point?